### PR TITLE
Add placeholder for release notes

### PR DESCRIPTION
**Pull request by maximiliankolb**
Refs https://github.com/theforeman/foreman-documentation/pull/2560

### CODE REVIEW

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 1]
```diff
@@ -7,6 +7,9 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+// If this section would be empty otherwise, uncomment the following line:
+// There are no upgrade warnings with Foreman {ProjectVersion}.
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:11]:
```suggestion
There are no upgrade warnings with Foreman {ProjectVersion}.
```

I'd say keep it uncommented. The first person to add a release note can then remove the line.

Comment (maximiliankolb) [reply guides/doc-Release_Notes/topics/foreman.adoc:11]: There is an upgrade warning in line 13.

**Comment by mjivraja** [comment]
My point is `Headline Features` looks empty. I guess it should have a placeholder too or some other treatments.

File: guides/doc-Release_Notes/topics/foreman.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/doc-Release_Notes/topics/foreman.adoc:11]: Right, I missed that. Sorry.

**Review by adamlazik1 — APPROVED** [review]

**Comment by adamlazik1** [comment]
> My point is `Headline Features` looks empty. I guess it should have a placeholder too or some other treatments.

I presume that there is going a be a feature for every single foreman release so we can be sure that it won't stay empty.

**Comment by maximiliankolb** [comment]
Yes, I agree; every release should have at least one headline feature. OK @mjivraja ?

**Comment by mjivraja** [comment]
> Yes, I agree; every release should have at least one headline feature. OK @mjivraja ?

I mean, can we add a placeholder at this point, the same as deprecations? Until we have a feature.

**Comment by maximiliankolb** [comment]
I have added a placeholder for all sections. Does this make sense to you @mjivraja and @adamlazik1 ?

**Review by mjivraja — APPROVED** [review]
LGTM

**Review by adamlazik1 — APPROVED** [review]

**Pull request merged**